src/engine/video/image.cpp

```diff
@@ -189,18 +184,16 @@ void ImageDescriptor::SetVertexColors(const Color &tl, const Color &tr, const Co
     _color[2] = bl;
     _color[3] = br;
 
-    if(IsFloatEqual(tl[3], 1.0f) && IsFloatEqual(tr[3], 1.0f) && IsFloatEqual(bl[3], 1.0f) && IsFloatEqual(br[3], 1.0f))
-        _blend = false;
-    else
-        _blend = true;
+    _blend = !(
+      IsFloatEqual(tl[3], 1.0f) && IsFloatEqual(tr[3], 1.0f) &&
+      IsFloatEqual(bl[3], 1.0f) && IsFloatEqual(br[3], 1.0f)
+    );
 
-    if(tl == tr && tl == bl && tl == br)
-        _unichrome_vertices = true;
-    else
-        _unichrome_vertices = false;
+    _unichrome_vertices = tl == tr && tl == bl && tl == br;
```

Comment:
Could you add parenthesis? Like:
`_unichrome_vertices = (tl == tr && tl == bl && tl == br);`

Otherwise, I found it hard to read even if perfectly valid.